Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -9,9 +9,9 @@
 
 ## 🦉 Domain expertise
 
-- I am a well-rounded developer, architect and problem solver
-- I am an expert in my team's domain
-- I am comfortable diving head-first into moderately risky situations with relatively little upfront information and oversight
+- I am a well-rounded developer, architect and problem solver.
+- I am an expert in my team's domain.
+- I am comfortable diving head-first into moderately risky or unknown situations with relatively little upfront information and oversight.
```

Comment:
```suggestion
- I am comfortable diving head-first into moderately risky or uncertain situations with relatively little upfront information and oversight.
```

(I'm assuming you meant "uncertain" here, from [this comment](https://whimsical.com/2024-q2-post-calibration-role-ladder-refinement-workshop-9xs7MapSCHDk6xwz58Veuk@2bsEvpTYSt1Hj7HE2CR1XBxoEVm7YkQ49oN))